<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialog_mirror</class>
 <widget class="QDialog" name="dialog_mirror">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>428</width>
    <height>296</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Choose a Download Server</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout">
   <property name="margin">
    <number>9</number>
   </property>
   <property name="spacing">
    <number>6</number>
   </property>
   <item row="0" column="2">
    <widget class="QWidget" name="buttons_frame" native="true">
     <layout class="QGridLayout">
      <property name="margin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>6</number>
      </property>
      <item row="0" column="0">
       <widget class="QWidget" name="edit_buttons_frame" native="true">
        <layout class="QGridLayout">
         <property name="margin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>6</number>
         </property>
         <item row="3" column="0">
          <spacer>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>113</width>
             <height>16</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="0">
          <widget class="QPushButton" name="button_remove">
           <property name="text">
            <string>Remove</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="button_edit">
           <property name="text">
            <string>Edit</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QPushButton" name="button_add">
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="button_find_server">
        <property name="text">
         <string>_Select Best Server</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="2">
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>133</width>
       <height>21</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" rowspan="2" colspan="2">
    <widget class="QTreeWidget" name="treeview">
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="protocol_label">
     <property name="text">
      <string>Protocol:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QComboBox" name="combobox"/>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>dialog_mirror</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>dialog_mirror</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
